src/hooks/useNotificationTypes.tsx

```diff
@@ -680,6 +684,9 @@ export const notificationTypes: NotificationTypeConfig[] = [
           const existingOrder = allOrders?.find((order) => order.id === localCancel.orderId);
           if (!existingOrder) return;
 
+          // skip if this is from a cancel all operation and isn't an error
+          if (localCancel.isSubmittedThroughCancelAll && !localCancel.errorParams) return;
```

Comment:
when user clicks cancel all, we won't show a new toast per cancel. we'll just have one notif, and only show a toast for the cancel when there was an error for that specific operation